runner/Dockerfile

```
@@ -25,12 +25,50 @@ FROM build AS binary
+    mkdir -p /usr/local/bin && \
+    printf '%s\n' '#!/bin/sh' "exec \"$HM_REAL\" \"\$@\"" > /usr/local/bin/home-manager && \
+    chmod +x /usr/local/bin/home-manager
+
+RUN nix-env -e bash-interactive bash || true
+
+RUN mkdir -p /root/.config/home-manager
+COPY ./nix/home.nix /root/.config/home-manager/home.nix
+
+RUN home-manager switch
+
+RUN nix shell nixpkgs#gnused --command sh -c '\
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)